docs/rules/object-curly-newline.md

```diff
@@ -1,17 +1,17 @@
-# enforce consistent line breaks inside braces (object-curly-newline)
+# enforce consistent line breaks after opening and before closing braces (object-curly-newline)
 
 A number of style guides require or disallow line breaks inside of object braces and other tokens.
 
 ## Rule Details
 
-This rule enforces consistent line breaks inside braces of object literals or destructuring assignments.
+More precisely, the rule requires or disallows a line break between { and its following token, and between } and its preceding token of object literals or destructuring assignments.
```

Comment:
```suggestion
This rule requires or disallows a line break between `{` and its following token, and between `}` and its preceding token of object literals or destructuring assignments.
```

Since this is now the first sentence in this section, it wouldn't be clear what "More precisely" refers to.

Also added backticks around `{` and `}`